src/convertToObject.js

```diff
@@ -6,7 +6,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((value) => value.trim())
+    .filter((value) => value !== '')
+    .reduce((prev, string) => {
+      return {
+        ...prev,
+        [string.split(':')[0].trim()]: string.split(':')[1].trim(),
+      };
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes the code much more readable. Align the methods `.split`, `.map`, `.filter`, and `.reduce` so they start at the same column.